build/Dockerfile.base

```diff
@@ -30,69 +30,70 @@ ARG DEBIAN_FRONTEND="noninteractive"
+   time \
+   tini \
+   && apt-get -y upgrade \
+   && su - easy_infra -c "mkdir -p /home/easy_infra/.ssh" \
+   && su - easy_infra -c "touch /home/easy_infra/.ssh/known_hosts" \
+   && echo "# START preloaded known_hosts as of $(date)" >> /home/easy_infra/.ssh/known_hosts \
+   && apt-get -y install --no-install-recommends ssh \
+   && ssh-keyscan gitlab.com \
+   github.com \
+   bitbucket.org \
+   ssh.dev.azure.com \
+   git-codecommit.us-east-1.amazonaws.com \
+   git-codecommit.us-east-2.amazonaws.com \
+   git-codecommit.us-west-1.amazonaws.com \
+   git-codecommit.us-west-2.amazonaws.com \
+   >> /home/easy_infra/.ssh/known_hosts \
+   && echo "# END preloaded known_hosts as of $(date)" >> /home/easy_infra/.ssh/known_hosts \
+   && apt-get remove -y ssh \
+   && mkdir -p /root/.ssh \
+   && touch /root/.ssh/known_hosts \
+   && cp /home/easy_infra/.ssh/known_hosts /root/.ssh/known_hosts \
+   && if [ "${TRACE}" = "true" ]; then \
+   apt-get -y install --no-install-recommends libcap2-bin \
+   strace \
+   ; fi \
+   && su easy_infra -c "mkdir -p /home/easy_infra/.local/bin/" \
+   && apt-get -y install --no-install-recommends unzip \
+   && curl -L https://releases.hashicorp.com/consul-template/${CONSUL_TEMPLATE_VERSION#v}/consul-template_${CONSUL_TEMPLATE_VERSION#v}_linux_${BUILDARCH}.zip -o /usr/local/bin/consul-template.zip \
+   && unzip /usr/local/bin/consul-template.zip -d /usr/local/bin/ \
+   && rm -f /usr/local/bin/consul-template.zip \
+   && chmod 0755 /usr/local/bin/consul-template \
+   && curl -L https://releases.hashicorp.com/envconsul/${ENVCONSUL_VERSION#v}/envconsul_${ENVCONSUL_VERSION#v}_linux_${BUILDARCH}.zip -o /usr/local/bin/envconsul.zip \
+   && unzip /usr/local/bin/envconsul.zip -d /usr/local/bin/ \
+   && rm -f /usr/local/bin/envconsul.zip \
+   && chmod 0755 /usr/local/bin/envconsul \
+   && apt-get remove -y unzip \
+   && apt-get install -y --no-install-recommends cmake build-essential flex bison libyaml-dev libssl-dev \
+   && git clone https://github.com/fluent/fluent-bit --depth 1 --branch ${FLUENT_BIT_VERSION} \
+   && cd fluent-bit/build \
+   && cmake ../ && make && make install \
+   && cd ../.. \
+   && rm -rf fluent-bit \
+   && apt-get remove -y cmake build-essential flex bison libssl-dev \
+   && echo "source /functions" >> /home/easy_infra/.bashrc \
+   && echo "source /functions" >> /root/.bashrc \
+   && apt-get clean autoclean \
+   && apt-get -y autoremove \
+   # Must be after the autoremove for fluent-bit to work
+   && apt-get remove -y libyaml-dev \
+   && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/* /tmp/* /var/tmp/* /var/log/* /var/cache/debconf/*-old \
+   && touch /var/log/easy_infra.log /var/log/fluent-bit.log /var/log/40-set_default_terraform_version.log /var/log/50-terraform_dynamic_version_use.log /var/log/clone.log /var/log/clone.err.log \
+   && mkdir /iac \
+   && chown -R easy_infra: /var/log/*.log /iac \
+   && find /etc/ssh/ -type f -name '*key' -exec rm {} +
```

Comment:
There may be a better way to do this once we nail down how they're being brought in (my guess is around the `apt-get install ssh` / `ssh-keyscan` section) such as using a cleanup feature, but if not we should at least put the `find` right next to the command that's creating the unwanted files